collections/_article/dev-snapshot-godot-4-4-beta-1.md

```diff
@@ -0,0 +1,291 @@
+
+Not content to stop there, Lander-vr also significantly improved the editor UX for ReflectionProbe and VoxelGI by making their gizmos less intrusive (in [GH-99920](https://github.com/godotengine/godot/pull/99920), [GH-99969](https://github.com/godotengine/godot/pull/99969) and [GH-100370](https://github.com/godotengine/godot/pull/100370)).
+
+![Sponza with reflective floor](/storage/blog/godot-4-4-beta-1/reflection-probes.webp)
+
+After many requests and years of patient waiting from many within the community, we have finally re-introduced vertex shading which is both a significant performance optimization and and integral part of recreating PSX-style graphics ([GH-83360](https://github.com/godotengine/godot/pull/83360)). This turned out to be a significant endeavor for ywmaa who enlisted the help of veteran rendering contributor Clay John to get the feature of the finish line.
+
+Currently when trying to run Godot with the Forward+ or Mobile backend on a device that doesn’t support Vulkan, D3D12, or Metal, the engine will provide the user with an OS alert notifying them that they don’t have support for the needed graphics API and they need to try again with the Compatibility backend. This alert has proven to be confusing for users and the process of opening the scene ends up being cumbersome.
+
+Now with [GH-97142](https://github.com/godotengine/godot/pull/97142), first contribution from SheepYhangCN, the engine will automatically fall back to using OpenGL (the Compatibility backend) when the other backends are not available. This should provide the smoothest possible experience for users on older devices.
+
+Rendering contributors have been relentless in identifying and implementing performance enhancements this release cycle. Many areas of the renders, both 2D and 3D have been optimized and we expect that many users will notice an improvement to performance this release cycle. Especially for 2D lighting, rendering on mobile devices, and general 2D rendering. 
+
+### XR
+Thanks to Godot’s unique feature of it's editor being made with the engine itself, we’ve been able to bring the Godot editor to unconventional places, such as the web and Android devices. Building upon the latter, Fredia Huya-Kouadio completed the proof of concept started by Bastiaan Olij years ago, to add support for using the Android editor on XR devices using OpenXR ([GH-96624](https://github.com/godotengine/godot/pull/96624))! You can test the current version by sideloading the APK, currently supported on Meta Quest 3 or Quest Pro.
```

Comment:
I don't know if we're going to build an APK for it as part of beta1, but the PR adding support for the XR editor on Pico was merged (https://github.com/godotengine/godot/pull/101387). If we aren't making a build, then probably not worth mentioning until we are. But if we are, we should add a note here pointing out the two separate APKs: one for Meta and one for Pico.

Reply:
I think at the very least we should change `currently supported on Meta Quest 3 or Quest Pro.` to `currently supported on Meta Quest 3 or Quest Pro, and soon PICO 4 Ultra.` or something of that nature (and maybe refer to the PR that was just merged).

Reply:
I changed the whole last sentence to "The XR editor is currently available on the Meta Quest 3 and Quest Pro through the [Horizon Store](https://www.meta.com/experiences/godot-game-engine/7713660705416473/). Support for the PICO 4 Ultra is also in progress and will be available soon."

The previous text was pulled from the dev2 blog post which was made before https://godotengine.org/article/godot-editor-horizon-store-early-access-release/